openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -20,16 +21,24 @@
 from openedx.core.lib.api.view_utils import add_serializer_errors
 
 from ..errors import (
-    AccountUpdateError, AccountValidationError, AccountUsernameInvalid, AccountPasswordInvalid,
-    AccountEmailInvalid, AccountUserAlreadyExists,
+    AccountUpdateError, AccountValidationError,
+    AccountDataBadLength, AccountDataBadType,
+    AccountUsernameInvalid, AccountPasswordInvalid, AccountEmailInvalid, AccountCountryInvalid,
+    AccountUserAlreadyExists, AccountUsernameAlreadyExists, AccountEmailAlreadyExists,
     UserAPIInternalError, UserAPIRequestError, UserNotFound, UserNotAuthorized
 )
 from ..forms import PasswordResetFormNoActive
 from ..helpers import intercept_errors
 
 from . import (
-    EMAIL_MIN_LENGTH, EMAIL_MAX_LENGTH, PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH,
-    USERNAME_MIN_LENGTH, USERNAME_MAX_LENGTH
+    EMAIL_BAD_LENGTH_MSG, USERNAME_BAD_LENGTH_MSG,
+    EMAIL_BAD_TYPE_MSG, PASSWORD_BAD_TYPE_MSG, USERNAME_BAD_TYPE_MSG,
+    EMAIL_CONFLICT_MSG, USERNAME_CONFLICT_MSG,
+    EMAIL_INVALID_MSG,
+    EMAIL_MIN_LENGTH, PASSWORD_MIN_LENGTH, USERNAME_MIN_LENGTH,
+    EMAIL_MAX_LENGTH, PASSWORD_MAX_LENGTH, USERNAME_MAX_LENGTH,
+    PASSWORD_CANT_EQUAL_USERNAME_MSG, PASSWORD_EMPTY_MSG, PASSWORD_BAD_MIN_LENGTH_MSG, PASSWORD_BAD_MAX_LENGTH_MSG,
+    REQUIRED_FIELD_COUNTRY_MSG, REQUIRED_FIELD_CONFIRM_EMAIL_MSG, REQUIRED_FIELD_NAME_MSG
```

Comment:
I can do that, yeah, but I'm just worried that the resulting column would be so huge that it'd become the biggest existing in the codebase! I think it would be a more reasonable move to do `accounts.X` instead, though, because then we wouldn't have to deal with this mess of a list for adding/deleting things (and I've had to do that several times, and it wasn't fun).